### PR TITLE
Only run 'hwloc_testsuite' explicitly based on HPC variable

### PR DESCRIPTION
**Pull request by okurz**
Needs to be added explicitly to scheduling.

### CODE REVIEW

**Comment by okurz** [comment]
Follow-up to #3244, @Soulofdestiny, @tblume, @asmorodskyi please review ASAP -> https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/3244#issuecomment-314992928

**Comment by okurz** [comment]
@tblume would you like to join internal #testing IRC channel in the future?

**Comment by Soulofdestiny** [comment]
LGTM, 
@tblume we will create a dedicated test for that

**Comment by Soulofdestiny** [comment]
@tblume: Test lives here: https://openqa.suse.de/tests/1059388
is now scheduled because it depends on the image which needs to be created first

for follow-up we should discuss together with @asmorodskyi where else we can integrate this testsuite

File: products/sle/main.pm [hunk 1]
```diff
@@ -1355,6 +1353,9 @@ elsif (get_var('HPC')) {
             loadtest 'console/hostname';
             loadtest 'hpc/powerman';
         }
+        elsif (check_var('HPC', 'hwloc') && sle_version_at_least('12-SP2')) {
```
Comment (asmorodskyi) [inline products/sle/main.pm]: actually whole HPC module is supported only starting from SP2 so we can move this check into line 1338

**Comment by tblume** [comment]
@okurz ok, I've joined the testing channel
@Soulofdestiny  thanks, watching the test

**Comment by Soulofdestiny** [comment]
@tblume had to retrigger it, https://openqa.suse.de/tests/1059428  it's working ;)

**Comment by tblume** [comment]
@Soulofdestiny ah ok, I was just going to ask why status of test 1059388 was cancel :)

**Pull request merged**